ios/Classes/MapboxMapController.swift

```diff
@@ -1430,6 +1438,16 @@ class MapboxMapController: NSObject, FlutterPlatformView, MGLMapViewDelegate, Ma
         }
     }
 
+    func setMyLocationStyle(style: [String: String]) {
+        if let puckColor = UIColor(hexString: style["puckColor"] ?? "") {
+            userLocationAnnotationViewStyle.puckFillColor = puckColor
+        }
+        if let pulsingColor = UIColor(hexString: style["pulsingColor"] ?? "") {
+            userLocationAnnotationViewStyle.haloFillColor = pulsingColor
+        }
+        mapView.updateUserLocationAnnotationView()
```

Comment:
@felix-ht  can you take a look?
`mapView.updateUserLocationAnnotationView()` cannot help refresh new color after setState or hot reload